### PR TITLE
ft(S3C-3751): Add GetBucketEncryption route

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: lib/policyEvaluator/utils/actionMaps.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const sharedActionMap = {
     bucketGet: 's3:ListBucket',
     bucketGetACL: 's3:GetBucketAcl',
     bucketGetCors: 's3:GetBucketCORS',
+    bucketGEncryption: 'se:GetBucketEncryption',
```
Comment (jonathan-gramain) [inline lib/policyEvaluator/utils/actionMaps.js]: typos on this line

File: lib/policyEvaluator/utils/actionMaps.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ const sharedActionMap = {
     bucketGet: 's3:ListBucket',
     bucketGetACL: 's3:GetBucketAcl',
     bucketGetCors: 's3:GetBucketCORS',
+    bucketGEncryption: 's3:GetBucketEncryption',
```
Comment (jonathan-gramain) [inline lib/policyEvaluator/utils/actionMaps.js]: `bucketGEncryption` -> `bucketGetEncryption`?

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**